### PR TITLE
[IMP] depreciation methods doc

### PR DESCRIPTION
**Pull request by fhe-odoo**

### CODE REVIEW

File: accounting/payables/supplier_bills/assets.rst [hunk 1]
```diff
@@ -133,15 +133,15 @@ rather than $ 4,000.00. Consequently, the last entry is also lower and has an am
...
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The **Linear Depreciation Method** divides the initial Depreciable Value by the number of
+The **Straight Line Depreciation Method** divides the initial Depreciable Value by the number of
 depreciations planned. All depreciation entries have the same amount.
 
-The **Degressive Depreciation Method** multiplies the Depreciable Value by the **Degressive Factor**
-for each entry. Each depreciation entry has a lower amount than the previous entry.
+The **Declining Depreciation Method** multiplies the Depreciable Value by the **Declining Factor**
+for each entry. Each depreciation entry has a lower amount than the previous entry. The last depreciation entry doesn't use the declining factor but instead has an amount corresponding to the balance of the depreciable value so that it reaches $0 by the end of the specified duration.
```
Comment (kbr-odoo) [inline accounting/payables/supplier_bills/assets.rst:140]: break line before 100 characters :)

**Review by kbr-odoo — APPROVED** [review]

**Comment by kbr-odoo** [comment]
> _No description provided._

Hello, Fabrice!
Thank you for your contribution.

Before merging, I'd like to suggest you improving the title to something like [IMP] accounting: depreciation methods update,
and to add a message with a quick explanation of your commit. That way, the reason for pull requests can be easy and quickly identified.

Thank you! :)
Keli

**Comment by jcs-odoo** [comment]
@kbr-odoo @fhe-odoo 
I took the liberty to change the commit's title + breaking the line at 100th char, as recommended by Keli, so we can merge this :)
I believe this commit was created directly on GitHub so I'm not sure how possible/easy it is to modify it...

Cheers,
Jonathan

File: accounting/payables/supplier_bills/assets.rst [hunk 2]
```diff
@@ -133,15 +133,17 @@ rather than $ 4,000.00. Consequently, the last entry is also lower and has an am
...
+for each entry. Each depreciation entry has a lower amount than the previous entry. The last
+depreciation entry doesn't use the declining factor but instead has an amount corresponding to the
+balance of the depreciable value so that it reaches $0 by the end of the specified duration.
 
-The **Accelerated Degressive Depreciation Method** uses the Degressive Method, but with a minimum
-Depreciation equal to the Linear Method. This method ensures a fast depreciation at the beginning,
-followed by a constant one afterward.
+The **Declining Then Straight Line Depreciation Method** uses the Declining Method, but with a
+minimum Depreciation equal to the Straight Line Method. This method ensures a fast depreciation
+at the beginning, followed by a constant one afterwards.
```
Comment (jcs-odoo) [inline accounting/payables/supplier_bills/assets.rst:146]:
```suggestion
at the beginning, followed by a constant one afterward.
```

Without "s" in american English.

**Pull request merged**